checkov/terraform/checks/graph_checks/aws/SGAttachedToResource.yaml

```diff
@@ -0,0 +1,40 @@
+metadata:
+  name: "Ensure that Security Groups are attached to an other resource"
```

Comment:
good shout @gruebel 
@rotemavni @GiladMark this will probably require a small change upstream in the docs and title of this one